tests/main-text.spec.ts

```diff
@@ -112,4 +112,70 @@ test.describe('main-text', () => {
       })
     });
   });
+
+  test.describe('format text', () => {
+    test('bold text', async ({ page }) => {
```

Comment:
You could I think generalise this into a private function, e.g...

```
async formatAndAssert(page: PageOrWhatever, format: string, expect: string): Promise<void> {
      const mainText = new MainText(page);
      await mainText.setText(content);
      const text = 'To Ganymede and Titan';
      await mainText.selectText(text);
      await mainText.formatText(format);
      await mainText.assertTextFormat(expect, text);
}

test('italic text', async ({ page }) => {
    await formatAndAssert(page, 'Italics', 'em');
}
```

Could help make it easier to maintain in the future.